client/src/screens/Patient/Messages/NewMessage/index.js

```diff
@@ -96,9 +108,28 @@ const NewMessage = (props) => {
           </Grid>
         </Grid>
 
-        <Typography variant="h5" color="textSecondary" gutterBottom>
-          Notify me if not read by Jan 1, 2020.
-        </Typography>
+        <Grid className={classes.dateInput}>
+          <MuiPickersUtilsProvider utils={DateFnsUtils}>
```

Comment:
We already added this provider into our app. So remove this one.

Reply:
Acknowledged

Reply:
Just use 
```
<KeyboardDatePicker
              required
              margin="dense"
              id="date-picker-dialog"
              label="Select Notification Date"
              format="dd/MM/yyyy"
              value={selectedDate}
              onChange={handleDateChange}
              minDate={currentDate}
            />
```